### PR TITLE
MWPW-153629: [Mini compare] Stack cards on mobile

### PR DESCRIPTION
**Pull request by Axelcureno**
Removes mobile styles for mini compare chart and instead stacks them all in one column.

Resolves: https://jira.corp.adobe.com/browse/MWPW-153629

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/docs/library/kitchen-sink/merch-card
- After: https://mwpw-153629--milo--adobecom.hlx.page/docs/library/kitchen-sink/merch-card
Dummy page for the PSI check: https://mwpw-155619-cards-overlapping-sorting--milo--mirafedas.hlx.page/

### CODE REVIEW

**Review by Roycethan — CHANGES_REQUESTED** [review]
@Axelcureno 
1)Some rows are having more empty spaces and not adjusted accordingly
https://main--cc--adobecom.hlx.live/creativecloud?milolibs=mwpw-153629--milo--adobecom
image
2) More empty spce before the CTA's
https://main--cc--adobecom.hlx.live/cz/creativecloud?milolibs=mwpw-153629--milo--adobecom
3)For Tablet : it should be two column layput per latest XD even in Individuals tab:
Need to adjust:
https://main--cc--adobecom.hlx.live/cz/creativecloud?milolibs=mwpw-153629--milo--adobecom
image

image

**Comment by Axelcureno** [comment]
@Roycethan I'm seeing different, can you try refreshing the page when switching to mobile / tablet? This extra space is added normally when window is resized and not refreshed, since we calculate height with js.

Screenshot 2024-07-31 at 5 03 32 PM
Screenshot 2024-07-31 at 5 02 48 PM

**Review by Roycethan — CHANGES_REQUESTED** [review]
All the above 3 issues are different and are reproducible even at page refresh.  #1  although happens when scrolling down - scroll down adding extra padding or disturbing the calculation.
#2 and #3 are reproducible
Let me know if more details to reproduce needed

File: libs/deps/mas/merch-card-all.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
+\f\r]`,M=/<(?:(!--|\/[^a-zA-Z])|(\/?[a-zA-Z][^>\s]*)|(\/?$))/g,Pe=/-->/g,Ne=/>/g,_=RegExp(`>|${ie}(?:([^\\s"'>=/]+)(${ie}*=${ie}*(?:[^
```
[the remark this answers is not in the document]
Comment (Roycethan) [reply libs/deps/mas/merch-card-all.js:2]: @Axelcureno Can you plz review/fix these ? thanks

**Review by Roycethan — APPROVED** [review]
Marked issues are resolved, approving, Needs to be verified post other reviews for regressions

**Review by bozojovicic — APPROVED** [review]

File: libs/features/mas/web-components/src/merch-card.js [hunk 2]
```diff
@@ -234,6 +236,10 @@ export class MerchCard extends LitElement {
         ];
     }
 
+    get isMobile() {
```
Comment (yesil) [inline libs/features/mas/web-components/src/merch-card.js:239]: can we move this in utils.js?

Comment (Axelcureno) [reply libs/features/mas/web-components/src/merch-card.js]: Done

File: libs/deps/mas/merch-card-all.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
+\f\r]`,P=/<(?:(!--|\/[^a-zA-Z])|(\/?[a-zA-Z][^>\s]*)|(\/?$))/g,Le=/-->/g,Ue=/>/g,_=RegExp(`>|${se}(?:([^\\s"'>=/]+)(${se}*=${se}*(?:[^
```
[the remark this answers is not in the document]
Comment (mirafedas) [reply libs/deps/mas/merch-card-all.js:2]: @Axelcureno I added `libs/deps/mas` to `.github/codeql/codeql-config.yml`, but my PRs are not merged yet, you can add it as well to make this check pass

**Review by 3ch023 — APPROVED** [review]

**Review by Roycethan — CHANGES_REQUESTED** [review]
@Axelcureno Price Font sizes are of 24 px at https://main--cc--adobecom.hlx.live/creativecloud?milolibs=mwpw-153629--milo--adobecom
they are at about 16 px at Main, plz check thanks

**Comment by Roycethan** [comment]
> @Axelcureno Price Font sizes are of 24 px at https://main--cc--adobecom.hlx.live/creativecloud?milolibs=mwpw-153629--milo--adobecom they are at about 16 px at Main, plz check thanks

Fixed

**Review by Roycethan — CHANGES_REQUESTED** [review]
@Axelcureno 
AC: Suppress the empty rows

Still see Empty rows exists:
image

**Comment by Axelcureno** [comment]
> @Axelcureno AC: Suppress the empty rows
> 
> Still see Empty rows exists: image

I have removed the empty rows @Roycethan

**Comment by Roycethan** [comment]
> > @Axelcureno AC: Suppress the empty rows
> > Still see Empty rows exists: image
> 
> I have removed the empty rows @Roycethan

Fixed Thanks

**Pull request merged**